app/models/publish_intent.rb

```diff
@@ -58,6 +58,12 @@ def self.cleanup_expired
     where(:publish_time.lt => PUBLISH_TIME_LEEWAY.ago).delete_all
   end
 
+  def base_path_without_root
```

Comment:
yes that'd be better, thanks - (was blindly c+ping methods). @dougdroper would not be happy with the `return nil` there